shardingsphere-test/shardingsphere-pipeline-test/src/test/java/org/apache/shardingsphere/data/pipeline/core/util/PipelineContextUtil.java

```diff
@@ -159,8 +159,8 @@ public static MigrationJobItemContext mockMigrationJobItemContext(final Migratio
         PipelineProcessConfiguration processConfig = mockPipelineProcessConfiguration();
         MigrationProcessContext processContext = new MigrationProcessContext(jobConfig.getJobId(), processConfig);
         int jobShardingItem = 0;
-        TaskConfiguration taskConfig = new MigrationJobAPIImpl().buildTaskConfiguration(jobConfig, jobShardingItem, processConfig);
-        return new MigrationJobItemContext(jobConfig, jobShardingItem, null,
+        TaskConfiguration taskConfig = new MigrationJobAPIImpl().buildTaskConfiguration(jobConfig, jobShardingItem, processConfig, processContext.getWriteRateLimitAlgorithm());
+        return new MigrationJobItemContext(jobConfig, jobShardingItem, null, 
                 processContext, taskConfig, new DefaultPipelineDataSourceManager());
```

Comment:
The last blank character at end of line could be removed. And it's better to merge the last two lines into one line.